code/game/objects/items.dm

```diff
@@ -793,9 +824,9 @@ GLOBAL_DATUM_INIT(welding_sparks, /mutable_appearance, mutable_appearance('icons
 		filters -= outline_filter
 		outline_filter = null
 
-// Returns a numeric value for sorting items used as parts in machines, so they can be replaced by the rped
+/// Returns a numeric value for sorting items used as parts in machines, so they can be replaced by the RPED
 /obj/item/proc/get_part_rating()
-	return 0
+	return FALSE
```

Comment:
Says numeric value in the comment, dont change this to FALSE

Reply:
I am extremely smart.